examples/vue/src/stories/index.js

```diff
@@ -96,4 +98,15 @@ storiesOf('Addon Notes', module)
   ))
   .add('with some bold text', withNotes({ notes: 'My notes on some bold text' })(() => ({
     render: h => h('div', [h('strong', ['A very long text to display'])])
-  })));
\ No newline at end of file
+  })));
+
+
+  storiesOf('Addon Knobs', module)
+    .addDecorator(withKnobs)
```

Comment:
Yeah, I'm trying to figure out how, since this function would require access the the right `context`-object (it needs to know which story it's called on).